llama.cpp

```diff
@@ -12618,14 +12618,15 @@ static int llama_decode_internal(
     std::vector<llama_seq_id *>            seq_id_arr;
     std::vector<std::vector<llama_seq_id>> seq_id;
 
+    // this indicates we are doing pooled embedding, so we ignore batch.logits and output all tokens
+    bool embed_pooled = cparams.embeddings && cparams.pooling_type != LLAMA_POOLING_TYPE_NONE;
```

Comment:
Naming this `embd_pooled` seems more inline with the usage of the `embd` identifier in `llama.cpp`:

```suggestion
    const bool embd_pooled = cparams.embeddings && cparams.pooling_type != LLAMA_POOLING_TYPE_NONE;
```

Reply:
yup, agreed.